test/k8s-integration/main.go

```diff
@@ -55,10 +56,11 @@ var (
 	saFile            = flag.String("service-account-file", "", "path of service account file")
 	deployOverlayName = flag.String("deploy-overlay-name", "", "which kustomize overlay to deploy the driver with")
 	doDriverBuild     = flag.Bool("do-driver-build", true, "building the driver from source")
-	useStagingDriver  = flag.Bool("use-staging-driver", false, "use GKE managed Filestore CSI driver for the tests")
+	useManagedDriver  = flag.Bool("use-gke-driver", false, "use GKE managed Filestore CSI driver for the tests")
 
 	// Test flags
 	testFocus = flag.String("test-focus", "External.Storage", "test focus for Kubernetes e2e")
+	parallel  = flag.Int("parallel", 4, "the number of parallel tests setting for ginkgo parallelism")
```

Comment:
let's keep it the same as current which is 3

Reply:
ok. Done.